docs/installation/migrate-to-xcp-ng.md

```diff
@@ -135,11 +135,23 @@ You need to rescan the SR where you new VHD file is, so it can be detected. It w
 
 There's two options, both requiring to export your Hyper-V VM disk in VHD format.
 
+### Exporting the VM disk
```

Comment:
Other titles around this one seem to be without "-ing".

Reply:
Fixed by https://github.com/xcp-ng/xcp-ng-org/pull/299/commits/2438c2de6177bedf7929c26c023d8fea419ea411